cadquery/assembly.py

```diff
@@ -451,6 +453,15 @@ def save(
             See :meth:`~cadquery.occ_impl.exporters.assembly.exportAssembly`.
         """
 
+        # Handle the export mode setting
```

Comment:
This looks like too much boilerplate and the cast is not needed

Reply:
It's modeled after the `exportType` code directly beneath it.